sci_bert/experiments/ner.jsonnet

```diff
@@ -0,0 +1,79 @@
+// Configuration for a named entity recognization model based on:
+//   Peters, Matthew E. et al. “Deep contextualized word representations.” NAACL-HLT (2018).
+{
```

Comment:
it would be good if all experiment config files are in one dir. We can move the one in allennlp_config here or move this to allennlp_config

Reply:
I missed that folder. will move there.